test/hypothesis_utils.py

```diff
@@ -106,9 +89,41 @@ def array_shapes(draw, min_dims=1, max_dims=None, min_side=1, max_side=None):
         st.integers(min_side, max_side), min_size=min_dims, max_size=max_dims
     ).map(tuple))
 
-"""Strategy for generating test cases for quantized tensors.
+
+"""Strategy for generating test cases for tensors.
 The resulting tensor is in float32 format.
 
+Args:
+    shapes
+Generates:
+    (X, w, b, g): Tensors of type `float32` of the following drawen shapes:
+        X: (`nbatch, iChannels, H, W`)
+        w: (`oChannels, iChannels // groups, kH, kW)
+        b: `(oChannels,)`
+        g: Number of groups the input is divided into
+"""
```

Comment:
Is the above comment description the correct one for this function? It only returns a tensor.